city_scrapers/spiders/chi_board_elections.py

```diff
@@ -0,0 +1,208 @@
+            'all_day': False,
+            'documents': [],
+            'sources': [{
+                'url': response.url,
+                'note': ''
+            }],
+        }
+        if "69" in next_meeting:
+            data['location'] = self._parse_location(response)
+        else:
+            raise ValueError("The address has changed.")
+
+        data['end'] = {
+            'date': data['start']['date'],
+            'time': None,
+            'note': '',
+        }
+
+        link = response.xpath("//a/@href").extract()[2]
+        data['documents'] = self._parse_documents(link)
+        data['status'] = self._generate_status(data)
+        data['id'] = self._generate_id(data)
+
+        yield data
+
+    def _prev_meetings(self, response):
+        """
+        Meetingdate regex first searches for the 3 types of hyphens that chi_board_elections
+        uses (they like switching it up), and then finds a year number,
+        and returns everything in between.
+        """
+
+        meetings = response.xpath("//a").extract()
+        prevdate = None
+        for meeting in meetings:
+            meeting.replace('\xa0', ' ')  # Gets rid of non-breaking space character
+            try:
+                meetingdate = re.search(r'(–|- |-)(.+[0-9]{4})', meeting).group(2)
+                while len(meetingdate) > 30:
+                    meetingdate = re.search(r'(–|- |-)(.+[0-9]{4})', meetingdate).group(2)
+                meetingdate.lstrip()
+                if prevdate != meetingdate:  # To acount for duplicates
+                    data = {
+                        '_type': 'event',
+                        'name': "Electoral Board",
+                        'event_description': "",
+                        'classification': COMMISSION,
+                        'start': self._parse_start(meetingdate, meeting),
+                        'all_day': False,
+                        'location': self._parse_location(response),
+                        'documents': [],
+                        'sources': [{
+                            'url': response.url,
+                            'note': ''
+                        }],
+                    }
+                    data['end'] = {
+                        'date': data['start']['date'],
+                        'time': None,
+                        'note': '',
+                    }
+                    if "Minutes" in meeting:
+                        minuteslink = re.search(r'="(.+)"', meeting).group(1)
+                        data["documents"] = self._parse_documents(minuteslink)
+                    data['status'] = self._generate_status(data)
+                    data['id'] = self._generate_id(data)
+                    yield data
+                if not self._different_time(meeting):  # To handle the odd 7 AM/7PM meetings
+                    prevdate = meetingdate
+                if "Nov. 13" in meetingdate:  # To handle the one date that falls outside of scraper
```

Comment:
Good catch on the behavior here. We'll probably want something more general though. We have an example of iterating through text nodes (anything not directly contained by a tag) in an older version of [`chi_teacherpension`](https://github.com/City-Bureau/city-scrapers/blob/f18c0e9297b6756686e7ca7c74668c104dbe9219/city_scrapers/spiders/chi_teacherpension.py#L36)

Reply:
The spider brings back video links now, but I when I tried to iterate through the text nodes, by using text() and span in XPATH, I got the text back, but no links, so I wasn't sure how to fix that

Reply:
Gotcha, yeah this one is a bit tricky. I got it to work locally with this:

```python
meetings = response.xpath("//a|//span/text()").extract()
for meeting in meetings:
  if not meeting.strip():
    continue # ignores any of the empty strings that are returned from /text()
  ...
```

Let me know if that helps! You might have to change up some other pieces that are expecting t to be a link and not text, and you'll need to remove the " - To be considered..." text